index.html

```diff
@@ -0,0 +1,342 @@
+					</div>
+
+				</div>
+			</section>
+			<!-- about section ends -->
+
+			<!-- javaScript Featured Speakers section starts -->
+			<section id="feature-speakers">
+				<div class="section-wrapper">
+					<h2 class="speakers-heading">
+						<span>Featured Speakers</span>
+						<span class="speakers-border"></span>
+					</h2>
+					<div class="speakers-wrapper" id='speakers-block'></div>
+				</div>
+			</section>
+			<!-- javaSctipt Ends -->
+
+			<div id='summit-section' class='summit-section'>
+				<div class="section-wrapper">
+					<p class="hello-world">"Hello! Sharing world"</p>
+					<h1 class="h1-line">
+						<span>Poleramma Jatara</span>
+						<span>Venkatagiri</span>
+						<span>2022</span>
+					</h1>
+					<div class="summary-block">
+						<p>
+							It's believed that this festival has a long time ago during the time of Venkatagiri rajas in
+							the mid-18th century local people believed that the goddess herself came and used to talk
+							with the rajas during the time of festival. It is usually held every year on 3rd Wednesday
+							and Thursday after Vinkayaka Chaturthi.
+						</p>
+					</div>
+					<div class="contact-details">
+						<p>Please contact us per Email for any further questions about Poleramma Jaatara
+							2022, Nellore District, Andhra Pradesh</p>
+						<p><a href='mailto:contact@pjatara.com'>contact@pjatara.com</a></p>
+
+					</div>
+				</div>
+			</div>
+
+			<div id='pj-logo-section' class='pj-logo-section'>
+				<div class="section-wrapper">
+					<h2>
+						Poleramma Jatara 2022 Logo
+					</h2>
+					<span class="title-border"></span>
+					<p>
+						Poleramma Jatara is celebrated with grandeur. Though Kalivelamma is the village goddess,
+						Poleramma Jatara, which is the custom of Rajas, became very
+					</p>
+					<figure class="logo">
+						<a href="#home-page">
+							<img src="logo/pj-logo.svg" alt="Poleramma Jatara" title="Poleramma Jatara" width="200">
+						</a>
+					</figure>
+				</div>
+			</div>
+
+			<div id='past-pics' class='pj-past-section'>
+				<div class="section-wrapper">
+					<h2>
+						See the past Poleramma Jaatara
+					</h2>
+					<span class="jatara-border"></span>
+					<p>
+						Poleramma Jatara is celebrated with grandeur. Though Kalivelamma is the village goddess.
+					</p>
+					<div class="pj-past-info">
+						<figure class="pj-past">
+							<a href="#">
+								<img src="jatara/pj-1.png" alt="Poleramma Jatara" title="Poleramma Jatara 2"
+									width="300" />
+								<div class="past-text">
+									<h3>2020</h3>
+									<p>Poleramma Jatara</p>
+								</div>
+
+							</a>
+
+						</figure>
+						<figure class="pj-past">
+							<a href="#">
+								<img src="jatara/pj-2.png" alt="Poleramma Jatara" title="Poleramma Jatara 2"
+									width="300" />
+								<div class="past-text">
+									<h3>2018</h3>
+									<p>Poleramma Jatara</p>
+								</div>
+
+							</a>
+
+						</figure>
+					</div>
+				</div>
+			</div>
```

Comment:
- [ ] As the previous reviewer mentioned you don't have an about page, kindly move your about page code to a new file, you can name it 'about.html' and add a top bar. nav bar, and footer there

![Screenshot_20220730_064108](https://user-images.githubusercontent.com/25227022/181876534-4f08568c-32c4-4669-a73d-922925583bcc.png)